### PR TITLE
[plugin.audio.navahang] v2.1.0

### PR DESCRIPTION
**Pull request by amirrammari**
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by amirrammari** [comment]
I don't know why the Travis CI build failed.
Please help me.
@Rechi

File: plugin.audio.navahang/addon.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>audio</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+
+    <summary lang="en_GB">Plugin for Navahang</summary>
+    <description lang="en_GB">Navahang is a media company with one of the largest online archives, dedicated to Persian music along with 24/7 Live Radio.</description>
+    <disclaimer lang="en_GB">This plugin is not endorsed by Navahang LLC</disclaimer>
+    
+    <summary lang="en_NZ">Plugin for Navahang</summary>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]: No need to add `en_NZ` if it is the same as `en_GB`.
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <description lang="en_NZ">Navahang is a media company with one of the largest online archives, dedicated to Persian music along with 24/7 Live Radio.</description>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <disclaimer lang="en_NZ">This plugin is not endorsed by Navahang LLC</disclaimer>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    
+    <summary lang="en_US">Plugin for Navahang</summary>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]: No need to add `en_US` if it is the same as `en_GB`.
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    <description lang="en_US">Navahang is a media company with one of the largest online archives, dedicated to Persian music along with 24/7 Live Radio.</description>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    <disclaimer lang="en_US">This plugin is not endorsed by Navahang LLC</disclaimer>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    
+    <summary lang="fa_af">افزونه برای نواهنگ</summary>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
    <summary lang="fa_AF">افزونه برای نواهنگ</summary>
```

File: plugin.audio.navahang/addon.xml [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    <description lang="fa_af">نواهنگ یک شرکت رسانه ای با یکی از بزرگترین آرشیو های آنلاین است که به موسیقی فارسی و همراه با ۲۴/۷ رادیو زنده اختصاص یافته است.</description>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
    <description lang="fa_AF">نواهنگ یک شرکت رسانه ای با یکی از بزرگترین آرشیو های آنلاین است که به موسیقی فارسی و همراه با ۲۴/۷ رادیو زنده اختصاص یافته است.</description>
```

File: plugin.audio.navahang/addon.xml [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    <disclaimer lang="fa_af">این افزونه توسط شرکت نواهنگ تایید نشده است</disclaimer>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
    <disclaimer lang="fa_AF">این افزونه توسط شرکت نواهنگ تایید نشده است</disclaimer>
```

File: plugin.audio.navahang/addon.xml [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+    
+    <summary lang="fa_ir">افزونه برای نواهنگ</summary>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]: No need to add `fa_IR` if it is the same as `fa_AF`.
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+    <description lang="fa_ir">نواهنگ یک شرکت رسانه ای با یکی از بزرگترین آرشیو های آنلاین است که به موسیقی فارسی و همراه با ۲۴/۷ رادیو زنده اختصاص یافته است.</description>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
```

File: plugin.audio.navahang/addon.xml [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+    <disclaimer lang="fa_ir">این افزونه توسط شرکت نواهنگ تایید نشده است</disclaimer>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]:
```suggestion
```

File: plugin.audio.navahang/default.py [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+from xbmcplugin import addDirectoryItem, endOfDirectory
+
+ADDON = xbmcaddon.Addon()
+ADDONNAME = ADDON.getAddonInfo('name')
+ADDONID = ADDON.getAddonInfo('id')
+ADDONVERSION = ADDON.getAddonInfo('version')
+CWD = ADDON.getAddonInfo('path').decode("utf-8")
+LANGUAGE = ADDON.getLocalizedString
+
+DataLink = 'http://navahang.me/~siavash/webservice/GetMp3?type=Featured&from=0&to=20'
```
Comment (Rechi) [inline plugin.audio.navahang/default.py]: This url doesn't look official to me.

File: plugin.audio.navahang/resources/language/resource.language.en_NZ/strings.po [hunk 14]
```diff
@@ -0,0 +1,29 @@
+# Kodi Media Center language file
```
Comment (Rechi) [inline plugin.audio.navahang/resources/language/resource.language.en_NZ/strings.po]: No need for `en_NZ` language file if it is the same as the `en_GB` one.

File: plugin.audio.navahang/resources/language/resource.language.en_US/strings.po [hunk 15]
```diff
@@ -0,0 +1,29 @@
+# Kodi Media Center language file
```
Comment (Rechi) [inline plugin.audio.navahang/resources/language/resource.language.en_US/strings.po]: No need for `en_US` language file if it is the same as the `en_GB` one.

File: plugin.audio.navahang/resources/language/resource.language.fa_ir/strings.po [hunk 16]
```diff
@@ -0,0 +1,29 @@
+# Kodi Media Center language file
```
Comment (Rechi) [inline plugin.audio.navahang/resources/language/resource.language.fa_ir/strings.po]: No need for `fa_IR` language file if it is the same as the `fa_AF` one.

File: plugin.audio.navahang/addon.xml [hunk 17]
```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.audio.navahang" name="Navahang" version="2.1.0" provider-name="Amir Ammari">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.beautifulsoup" version="3.2.1"/>
```
Comment (Rechi) [inline plugin.audio.navahang/addon.xml]: I don't see any code using this module.
```suggestion
```

**Pull request closed without merging**